src/compliance/mocks/compliance-requirements.mock.ts

```diff
@@ -24,3 +31,26 @@ export class MockComplianceRequirements {
 
   defaultTrustedClaimIssuers = [];
 }
+
+export const mockComplianceRequirements = createMock<ComplianceRequirements>({
+  requirements: [
+    {
+      id: new BigNumber(1),
+      conditions: [
+        {
+          type: ConditionType.IsPresent,
+          claim: {
+            type: ClaimType.Accredited,
+            scope: {
+              type: ScopeType.Identity,
+              value: 'Ox6'.padEnd(66, '0'),
```

Comment:
(note) if my consts PR gets merged, update these.

If this gets merged in first I will update in that PR.